pkg/kubectl/cmd/templates/command_groups.go

```diff
@@ -59,3 +66,25 @@ func AddAdditionalCommands(g CommandGroups, message string, cmds []*cobra.Comman
 	}
 	return append(g, group)
 }
+
+func CheckRequiredFlags(cmd *cobra.Command, _ []string) error {
+	flags := cmd.Flags()
+	missingFlagNames := []string{}
+
+	flags.VisitAll(func(flag *pflag.Flag) {
+		requiredAnnotation, found := flag.Annotations[cobra.BashCompOneRequiredFlag]
```

Comment:
@liggitt This is suggested in [spf13/cobra #206](https://github.com/spf13/cobra/issues/206). Also this will not do any harm to these annotations. Did not find a better way to address this.